libraries/common/components/Modal/index.jsx

```diff
@@ -11,13 +12,20 @@ import styles from './style';
  */
 const Modal = ({ children, classes }) => (
   <ConnectedReactPortal isOpened>
-    <div className={classNames(styles.container, classes?.container, 'common__modal')}>
-      <div className={classNames(styles.layout, classes?.layout)}>
-        <div className={classNames(styles.content, classes?.content)}>
-          {children}
+    <FocusTrap>
```

Comment:
as discussed, the `FocusTrap` causes issues when there are no child components that can be focused. since we already had some examples where this occurs, we should remove it for now.